openstack/imageservice/v2/imageimport/results.go

```diff
@@ -0,0 +1,32 @@
+package imageimport
+
+import "github.com/gophercloud/gophercloud"
+
+type commonResult struct {
+	gophercloud.Result
+}
+
+// GetResult represents the result of a get operation. Call its Extract method
+// to interpret it as ImportInfo.
+type GetResult struct {
+	commonResult
+}
+
+// ImportInfo represents information data for the Import API.
+type ImportInfo struct {
+	ImportMethods ImportMethods `json:"import-methods"`
```

Comment:
Fields in `results.go` (which can be thought of as "outputs") get standard types. In this case `string`.

Reply:
You are absolutely correct. I'm sorry!